src/v2/ion/IonResponseHelper.js

```diff
@@ -0,0 +1,60 @@
+import { _ } from 'okta';
+
+const getRemediationErrors = (res) => {
+  let errors = [];
+  let remediationErrors = res.remediation && 
+    res.remediation.value[0] && 
+    _.omit(res.remediation.value[0].value, (value) => {
+      return value.name === 'stateHandle';
```

Comment:
looks like we are trying to filter out `stateHandle`, instead of hardcoding the check on the `name` can we check for the `visible` property to be false?